internal/utils/utf8.go

```diff
@@ -0,0 +1,12 @@
+package utils
+
+import (
+	"golang.org/x/text/runes"
+	"golang.org/x/text/transform"
+)
+
+func SanitizeUTF8(input string) (string, error) {
```

Comment:
could you move it into internal/jptrace/sanitizer package instead? I am establishing it as the central location for model sanitizers in #7158